examples/grafana.build.sh

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env bash
+
+#
+# A rkt container pod for running a Statsd/Graphite/Grafana metrics stack.
+#
+# Start the containers with new data storage directory:
+#
+#   mkdir data
+#   sudo rkt run \
+#     --port=carbon:2003 \
+#     --port=statsd:8125 \
```

Comment:
Graphite has a minimum time-window resolution. In this case it is 10 seconds. If you report more than one value in a ten second window then only the last value is stored. E.g. suppose you were storing request latency values from say a ping, then all metric values reported in the ten second window are lost except for the last one.

Statsd is more like the finagle-stats reporting where you specify that the latency is a `stat` metric and the library takes care of reporting the average, count, and percentiles for all the metric values in the ten second window.

(The `stat` is a `timing` in Statsd terms: https://github.com/etsy/statsd/blob/master/docs/metric_types.md)

Main difference is Statsd is a service instead of a library.

Note, even if you reduced the Graphite window size to a second, then Graphite is still going to aggregate the values up anyway. So you end up with a single value for the ten second window. With Statsd you get additional numbers at the ten second resolution indicating percentiles and number of reported values.